.github/workflows/test.yaml

```diff
@@ -56,4 +66,4 @@ jobs:
         fi
         echo "kafka-acl-init completed successfully."
     - name: Test BOOM
-      run: cargo test --release
+      run: PATH=/opt/kafka/bin:$PATH cargo test --release
```

Comment:
perhaps worth putting that as an `env` var at the top of the github action? Fine here too, this is a nitpicking